### DISCUSSION TITLE
MayaUSD plugin won't load, specified procedure could not be found error

### DISCUSSION THREAD

**Discussion by LasagnaGuy** [Autodesk/maya-usd#3943] [Q&A] [answered]
Hi everybody,
I'm on a new fresh windows clean install, and for some reason the USD plugin won't load in both Maya 2024.2 and 2025.2. This exact same config was working before on the same machine, except it was windows 10.

I already did another clean reinstall of both Maya 24-25 without success, and cleaning the preferences also doesn't help.

here's the errors from the script editor:

// Error: file: C:/Program Files/Autodesk/Maya2025/scripts/startup/autoLoadPlugin.mel line 40: Unable to dynamically load : C:/Program Files/Autodesk/MayaUSD/Maya2025/0.29.0/mayausd/MayaUSD/plugin/adsk/plugin/mayaUsdPlugin.mll
// The specified procedure could not be found.
// Error: file: C:/Program Files/Autodesk/Maya2025/scripts/startup/autoLoadPlugin.mel line 40: The specified procedure could not be found.
//  (mayaUsdPlugin)

User is a local admin, not the live MS account. Here are the usr env and system env:

![env](https://github.com/user-attachments/assets/cfeec720-0a41-45b1-80ae-76fe463938d7)
![usr_env](https://github.com/user-attachments/assets/4689e865-a169-4d7a-82e1-0a81d02c227b)

BIfrost is also throwing errors here and there because of this, but the plugin it's working though, just not the usd part.

Any idea how to solve this?

**Comment by LasagnaGuy** [comment]
Not sure why this is happening, but for the record, I fixed the problem by keeping only the windows/system variables, and removing all the others (nvidia, crucial etc). That's quite strange, since all those variables looks quite harmless to me, also it never happened before, using the previous OS.
Now everything is working perfectly.

**Comment by seando-adsk** [comment, marked as answer]
Very often the MayaUsd plugin loading problems are caused by a conflict with either your PATH or PYTHONPATH. Usually there is another USD build in one of these paths. MayaUsd comes with its own USD build that the plugin is compiled compiled with. 

Sean

**Comment by LasagnaGuy** [comment]
You are right indeed: I just checked where the USD build could be, and lo and behold, MRV2 has a USD build exactly in that path C:\Program Files\mrv2-v1.2.8\bin
Thanks seando-adsk

**Comment by seando-adsk** [comment]
Glad you figured that out. I'll close this discussion then.
Thanks, Sean
